packages/protocol/test/genesis/test_config.json

```diff
@@ -80,10 +80,10 @@
     "SignalServiceProxy": "0x1000777700000000000000000000000000000007"
   },
   "param1559": {
-    "yscale": "358298803609133338137582400989",
-    "xscale": "1488514844",
-    "gasIssuedPerSecond": "12500000",
-    "gasExcess": "45450000000"
+    "yscale": "358298803609133338137582400989", // Q: do we need to change this?
+    "xscale": "1488514844", // Q: do we need to change this?
+    "gasIssuedPerSecond": "2000000", // If we assume the avg block time is 3s (is it reasonable?) , and each block is full (block.gasLimit=6m)
+    "gasExcess": "45450000000" // Q: do we need to change this?
```

Comment:
I think we need `gasExcessMax` here as it is part of the params:

```solidity

    struct EIP1559Params {
        uint64 basefee;
        uint64 gasIssuedPerSecond;
        uint64 gasExcessMax;
        uint64 gasTarget;
        uint64 ratio2x1x;
    }

```

Reply:
In general, we should/can use [TestTaiko1559Params.t.sol](https://github.com/taikoxyz/taiko-mono/blob/main/packages/protocol/test/Taiko1559Params.t.sol)  to tune the `gasExcessMax` and other parameters.

Reply:
> I think we need `gasExcessMax` here as it is part of the params:
> 
> ```solidity
>     struct EIP1559Params {
>         uint64 basefee;
>         uint64 gasIssuedPerSecond;
>         uint64 gasExcessMax;
>         uint64 gasTarget;
>         uint64 ratio2x1x;
>     }
> ```

Hmm i think we only these values which will be written into TaikoL2's storage slots here, so here should be those results which are calculated by the `TaikoL2.init(EIP1559Params)` method.

Reply:
https://github.com/taikoxyz/taiko-mono/pull/13815 We can upgrade the settings by upgrading the contract like this.